deep_translator/utils.py

```diff
@@ -1,3 +1,4 @@
 """
 utilities
 """
+# TODO: Discussion: Whats the purpsoe of this module?
```

Comment:
we can remove utils.py. When I started the project I thought I can write all utilities functions here but looks like it is not needed

Reply:
Removed